services/py-api/src/database/repository/participants_repository.py

```diff
@@ -38,8 +38,11 @@ async def create(
         self,
         input_data: ParticipantRequestBody,
         session: Optional[AsyncIOMotorClientSession] = None,
-        **kwargs: Dict[str, Any]
+        **kwargs: Dict[str, Any],
     ) -> Result[Participant, DuplicateEmailError | Exception]:
+
+        session_id = session.session_id["id"].hex() if session is not None else None
```

Comment:
```python
session_id = session.session_id["id"].hex() if session else None
```

Reply:
Alright!

Reply:
Fixed in 62ecace76fd209bffa772b0238adedf16bd38dc2